libs/expo/betterangels/src/lib/apollo/graphql/queries.ts

```diff
@@ -9,6 +9,7 @@ export const GET_NOTES = gql`
     notes(filters: $filters, pagination: $pagination, order: $order) {
```

Comment:
Hey @mikefeldberg 

It might be a silly question but how do I test this without entering Team's name? 
Your Add Team Section user story is coming up next (DEV-780) so shouldn't we the current ticket tested after DEV-780? Because I don't see the Team section atm with the QR code...

Unless there is a specific way of testing, please let me know. Thanks!

Reply:
@shiozakiyuko good call out, here. I should have included specific steps to test. in this case, since there's no UI to update interaction team, you could test by:
1. in app: create an interaction
2. in django admin: update that interaction's team
3. in app: view that interaction's summary screen